openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -103,22 +103,22 @@ <h3 class="message__title">{% trans "We could not save your progress" %}</h3>
 
                         <ul class="list list--actions">
                             <li class="list--actions__item">
-                                <button type="submit" class="action action--save submission__save is--disabled" aria-describedby="response__save_status__{{ xblock_id }}">
+                                <button type="submit" class="action action--save submission__save" aria-describedby="response__save_status__{{ xblock_id }}" disabled>
                                     {% trans "Save your progress" %}
                                 </button>
 
-                                <div id="response__save_status__{{ xblock_id }}" class="response__submission__label">
+                                <div id="response__save_status__{{ xblock_id }}" class="save__submission__label response__submission__label">
```

Comment:
Note that "Preview in LaTeX" (which I can't figure out how to enable on sandbox... I hacked the HTML file to make it visible on my devstack) has a text area with the class response__submission__label as well. They both use that class for styling of text beside the button. However, there is JS that is injecting a message about saving the response, and because the response__submission__label class was used, the message erroneously appeared beside both buttons.

Added save__submission__label so that the JS will only inject the message into the text area beside the Save button.